cmd/goal/defaultNetworkTemplate.json

```
@@ -0,0 +1,45 @@
+{
+    "Genesis": {
+        "NetworkName": "",
```

This came up in a chat on dev just now. Good thing to include here since we don't use "rewards" anymore.
```suggestion
        "NetworkName": "",
        "RewardsPoolBalance": 0
```